log4j-core/src/main/java/org/apache/logging/log4j/core/pattern/LevelPatternConverter.java

```diff
@@ -123,7 +120,7 @@ private static String left(final Level level, final int length) {
      */
     @Override
     public void format(final LogEvent event, final StringBuilder output) {
-        output.append(levelMap == null ? event.getLevel().toString() : levelMap.get(event.getLevel()));
+
```

Comment:
Is this supposed to be empty, or should this method be removed and handled by super?

Reply:
This method should be removed, it's effectively abstract but moving the base class from final -> abstract upset revapi.